src/angular-app/bellows/polyfills.browser.ts

```diff
@@ -1,24 +1,3 @@
 // Polyfills
 
-import 'core-js/es/array';
```

Comment:
I would love to get rid of corejs !  However, I don't remember talking about removing the polyfills in this file, so I think more testing would be necessary before we can remove this as well.  Probably safest to remove corejs and the polyfills.browser.ts in a separate PR.

Reply:
You got it, I have reverted that file and am working on the other PR